philipmeneghini/InstaConnect
# Frontend Utilizes Functional Components and Converted to Typescript

**Testing Instructions**

- Download dependencies on your local machine with `npm install`
- Launch application with `npm start`
- Test to make sure that the "Click" button still works
- It should return an error since the database and backend is being reconfigured in another PR
